clang/test/CIR/CodeGen/try-catch-dtors.cpp

```diff
@@ -339,3 +339,99 @@ void bar() {
 // CIR:  cir.store %[[V3]], %[[V1]] : !s32i, !cir.ptr<!s32i>
 // CIR:  cir.call @_ZN1AD2Ev(%[[V0]]) : (!cir.ptr<!ty_A>) -> () extra(#fn_attr)
 // CIR:  cir.return
+
+struct S {
```

Comment:
Please put these tests in `clang/test/CIR/CodeGen/throw.cpp` and also check for the LLVM output